unitex/src/fr/umlv/unitex/leximir/delas/ChooseDelas.java

```diff
@@ -0,0 +1,275 @@
+                    }
+                };
+
+                if (allDFoler.list(filter).length == 0) {
+                    JOptionPane.showMessageDialog(null, "No Delas was Found in : " + DictionaryPath.allDelas, "Information", JOptionPane.INFORMATION_MESSAGE);
+                } else {
+                    GlobalProjectManager.search(null).getFrameManagerAs(UnitexInternalFrameManager.class)
+                            .newEditorDelasDialog(all, null);
+                    this.setVisible(false);
+                }
+            } else {
+                int reply = JOptionPane.showConfirmDialog(null, "No Delas folder found in Dela folder \nCreate Delas folder?", title,
+                        JOptionPane.YES_NO_OPTION);
+                if (reply == JOptionPane.YES_OPTION) {
+                    allDFoler.mkdirs();
+                    if (allDFoler.exists()) {
+                        JOptionPane.showMessageDialog(null, "The folder created successfully !", "Information", JOptionPane.INFORMATION_MESSAGE);
+                    }
+
+                    GlobalProjectManager.search(null).getFrameManagerAs(UnitexInternalFrameManager.class)
+                            .newEditorDelasDialog(all, null);
+                    this.setVisible(false);
+                } else {
+                    this.jRadioAllDelas.setEnabled(false);
+                    this.jRadioBrowse.setSelected(true);
+                }
+            }
+        } else {
+            GlobalProjectManager.search(null).getFrameManagerAs(UnitexInternalFrameManager.class)
+                    .newEditorDelasDialog(all, new File(jTextField1.getText()));
+            this.setVisible(false);
+        }
+
+    }
+
+    private void jButtonBrowseActionPerformed(java.awt.event.ActionEvent evt) {
+        JFileChooser fileChooser = new JFileChooser();
+        FileNameExtensionFilter filter = new FileNameExtensionFilter("*.dic", "dic");
+        fileChooser.setFileFilter(filter);
+        fileChooser.setCurrentDirectory(new File(DictionaryPath.allDela));
+        int result = fileChooser.showOpenDialog(this);
+        if (result == JFileChooser.APPROVE_OPTION) {
+            File selectedFile = fileChooser.getSelectedFile();
+            jTextField1.setText(selectedFile.getAbsolutePath());
+            jRadioBrowse.setSelected(true);
+        }
+
+    }
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String args[]) {
+
+        try {
+            for (javax.swing.UIManager.LookAndFeelInfo info : javax.swing.UIManager.getInstalledLookAndFeels()) {
+                if ("Nimbus".equals(info.getName())) {
+                    javax.swing.UIManager.setLookAndFeel(info.getClassName());
+                    break;
+                }
+            }
+        } catch (ClassNotFoundException ex) {
+            java.util.logging.Logger.getLogger(ChooseDelas.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
```

Comment:
Is this really necessary ? Why ?

Reply:
It was Generated automatically by my IDE, I didn't pay attention to It, I'm deleting all those main functions in my classes